src/frontend/src/components/plugins/PluginContext.tsx

```diff
@@ -34,3 +39,37 @@ export type PluginContext = {
   theme: MantineTheme;
   colorScheme: MantineColorScheme;
 };
+
+export const useInvenTreeContext = () => {
+  const host = useLocalState.getState().host;
```

Comment:
Q: Why is the state invocated this way? I think a change in `host` would not be propagated with this kind of invocation

Reply:
You're right, normally that should be different. But I have copied it from the initial implementation from @SchrodingersGat , maybe he had something in mind with it? I have now changed it.